Trimble.Modus.Components/Controls/TMSpinner/TMSpinner.cs

```diff
@@ -16,14 +17,19 @@ public class TMSpinner : SKCanvasView
         private int minWidth = 42, minHeight = 42, animateTimerSeconds = 10;
         private float _sweepAngle = 180f;
         private float _rotationAngle = 0f;
-        private Timer _animationTimer;
+        private Timer _animationTimer;
+        private int _strokeWidth;
         #endregion
         #region Binding Properties
         public static readonly BindableProperty SpinnerTypeProperty =
             BindableProperty.Create(nameof(SpinnerType), typeof(SpinnerType), typeof(TMSpinner), defaultValue: SpinnerType.InDeterminate, propertyChanged: OnSpinnerTypeChanged);
 
         public static readonly BindableProperty SpinnerColorProperty =
-            BindableProperty.Create(nameof(SpinnerColor), typeof(SpinnerColor), typeof(TMSpinner), defaultValue: SpinnerColor.Primary, propertyChanged: OnSpinnerColorChanged);
+            BindableProperty.Create(nameof(SpinnerColor), typeof(SpinnerColor), typeof(TMSpinner), defaultValue: SpinnerColor.Primary, propertyChanged: OnSpinnerColorChanged);
+
+        public static readonly BindableProperty SpinnerSizeProperty =
```

Comment:
Can you make this internal? We should not expose this outside

Reply:
I see you have made the `SpinnerSize` as `internal`. But shouldn't make the bindable property too?

Reply:
Yes, modified it to internal